compose.js

```diff
@@ -89,7 +89,11 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
     if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    dstDescriptor.initializers.push.apply(dstDescriptor.initializers, srcDescriptor.initializers.filter(isFunction));
+    srcDescriptor.initializers.forEach(i => {
```

Comment:
Ok. Done. See the latest commit.
Also, the invitation was sent. Feel free to accept and merge the PR (use the "squash and merge" github button pls).